modules/identify-container-image-layer-that-introduces-vulnerabilities.adoc

```diff
@@ -11,8 +11,7 @@ Use the *Vulnerability Management* view to identify vulnerable components and th
 
 .Procedure
 
-. Navigate to the RHACS portal and click *Vulnerability Management* from the navigation menu.
-. Select an image from the *Top Riskiest Images* widget.
-. In the *Image* details view, select the *Dockerfile* tab under the *Image Findings* section.
-. In the *Dockerfile* tab under the *Image Findings* section, select the expand icon to see a summary of image components.
+. Navigate to the {product-title-short} portal and click *Vulnerability Management* from the navigation menu.
+. Select an image from the *Top Riskiest Images* widget or click the *Images* button at the top of the Dashboard and select an image.
+. In the *Image* details view, next to *Dockerfile*, select the expand icon to see a summary of image components.
```

Comment:
Besides the main Expand icon (triangle) to open the Dockerfile view, each text row of the file that adds components also has its own Expand icon, which you have to click to show the components added by that the command in that row.

(Note: this behavior has been this way for at least 3 years. It is not changed in this release.)

Reply:
The original said to only access the Vuln Management through the Top Riskiest Images - I added that you could click Images to go there also. Also removed "tab" language since the Dockerfile field isn't a separate tab. 

*each text row of the file that adds components also has its own Expand icon, which you have to click to show the components added by that the command in that row.*

Isn't that Step 4? I didn't make any changes to that part.